install/kubernetes/helm/subcharts/gateways/templates/deployment.yaml

```diff
@@ -54,6 +54,19 @@ spec:
             privileged: true
 {{- end }}
       containers:
+{{- if eq $key "istio-ingressgateway" }}
+{{- if $.Values.ingressGatewayAgent.enabled }}
```

Comment:
'agent' is a very generic and confusing name - in general security should stop using 'node agent' or other vague names. 

ingressSdsAgent, or simply ingressSds would work better.

Reply:
Renamed to ingress-sds.